tools/rosgraph/src/rosgraph/xmlrpc.py

```diff
@@ -76,9 +77,36 @@ def isstring(s):
     except NameError:
         return isinstance(s, str)
 
-class SilenceableXMLRPCRequestHandler(SimpleXMLRPCRequestHandler):
 
-    protocol_version = 'HTTP/1.1'
+def _check_linux_kernel(major, minor):
+    """Small helper version to check if the current Linux kernel is older than
```

Comment:
This function's purpose / return value is not clear from the function name.. maybe `_linux_kernel_at_most` instead? A `:returns:` notice in the docstring would also help clarify.

You could also make the fn just be called `support_http_1_1` and then heavily comment your logic inside that function.